src/types/Movie.ts

```diff
@@ -0,0 +1,7 @@
+export interface Movie {
+  title: string;
+  description?: string;
```

Comment:
Why description property is not required?
We have the description in each card

Reply:
it says so in the task description
"All the fields in the `movie` except the `description` are required"